HealthcareSystem/EventSourcingServiceTests/IntegrationTests/EvenStartSchedulingServiceTests.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using EventSourcingService.DTO;
+using EventSourcingService.Model;
+using EventSourcingService.Model.Enum;
+using EventSourcingService.Repository;
+using EventSourcingService.Service;
+using Microsoft.EntityFrameworkCore;
+using Xunit;
+
+namespace EventSourcingServiceTests.IntegrationTests
+{
+    public class EvenStartSchedulingServiceTests
+    {
+        private EventStorePatientStartSchedulingService SetupEventStartService()
+        {
+            EventDataSeeder dataSeeder = new EventDataSeeder();
+            DbContextOptionsBuilder<EventSourcingDbContext> builder = new DbContextOptionsBuilder<EventSourcingDbContext>(); ;
+            DbContextOptions<EventSourcingDbContext> options;
+            EventSourcingDbContext context;
+            builder.UseInMemoryDatabase(Guid.NewGuid().ToString());
+            options = builder.Options;
+            context = new EventSourcingDbContext(options);
+
+            dataSeeder.SeedAll(context);
+
+            var eventRepo = new DomainEventRepository<PatientStartSchedulingEvent>(context);
+            return new EventStorePatientStartSchedulingService(eventRepo);
+        }
+
+        [Fact]
+        public void SuccessAddNewStartEvent()
+        {
+            EventStorePatientStartSchedulingService eventStartService = SetupEventStartService();
+            PatientStartSchedulingEvent startEvent = eventStartService.Add(new PatientStartSchedulingEventDTO() { TriggerTime = DateTime.Now, UserAge = 35, UserGender = Gender.Female});
+
+            Assert.True(eventStartService.Contain(startEvent.Id));
```

Comment:
Did you add these methods just for tests? I don't like that at all, just do a get all here and assert it contains what you wanted, that would be less weird. Or check though the DbContext that it contains what you want, although that would be questionable in terms of testing observable behavior vs implementation details.